src/app/components/champion/ChampionView.tsx

```diff
@@ -66,6 +71,10 @@ export const ChampionView: FC<ChampionProps> = ({champion, callback, championsLi
         }
     }))(Tooltip)
 
+    function cleanDescription(description:string) {
+        return description.replace(/<[^>]*>/g, '')
```

Comment:
A comment to justify this kind of codes pls.

Reply:
regex are those thing that you always forget their purposes after a good night sleep

Reply:
Added the reason in comment as well as a special thank you to Riot Games.